app/services/forest_liana/has_many_getter.rb

```diff
@@ -5,6 +5,9 @@ def initialize(resource, association, params)
       @association = association
       @params = params
       @field_names_requested = field_names_requested
+      @current_collection = ForestLiana.apimap.find do |collection|
+        collection.name.to_s == association_table_name
+      end
```

Comment:
You should create a method in the parent class `BaseGetter` for this. The implementation would be at a single place for the `ResourcesGetter` and the `HasManyGetter`.